### ISSUE TITLE
[Feature Request] Start Obsidian on Startup in "Silent Mode" (Start Obsidian without showing that it loads..)

### ISSUE THREAD

**Issue by CrackheadoooObsidianUserooo** [dragonwocky/obsidian-tray#38]
# Following "Problem"

- When I start my PC, Obsidian starts aswells but always opens the whole Obsidian. 
I would like if Obsidian loads silently until it is completly loaded

- When Obsidian start and then I want to open It for the first time, it still loads somethin (workspaces)
Is it possible for Obsidian to "fully load"?

**Thx for reading :)**

**Comment by DesertDwarf** [comment]
For your first item, there are two hide options: `Hide on launch` and `Hide taskbar icon`. Those both work for me. Do they not work for you?

For your second item, when we install a Community plugin, we are installing it in the vault we are working on at the moment. Any other vaults you have on your system will need their own installation of Tray added to them in order for them to also load on startup.

In other words, each vault is self-contained, complete, and knows nothing about other vaults.

Or, did I misunderstand what you meant by "workspace"?

**Comment by ivkom** [comment]
> # Following "Problem"
> * When I start my PC, Obsidian starts aswells but always opens the whole Obsidian.
>   I would like if Obsidian loads silently until it is completly loaded
> * When Obsidian start and then I want to open It for the first time, it still loads somethin (workspaces)
>   Is it possible for Obsidian to "fully load"?
> 
> **Thx for reading :)**
Check autorun apps.
I had a similar problem. For some reason, two copies of Obsidian were loaded.
One through the registry (obsidian.exe), and the other through the startup folder (obsidian.lnk). I deleted the lnk and everything became normal.

**Comment by FaintWhisper** [comment]
> For your first item, there are two hide options: `Hide on launch` and `Hide taskbar icon`. Those both work for me. Do they not work for you?

In my case, I have the `Hide on launch` option enabled, and initially, I can observe the Obsidian window along with the loading bar on login. However, it is automatically hidden after a few seconds.

I suspect it is hidden the moment the Tray is loaded. So, if you have a high number of plugins installed, as I do, or if you have a slow machine, you may see the Obsidian loading bar for a few seconds. While this is not a major inconvenience for me, it can be a bit annoying. I would prefer it to remain hidden from the beginning. 

Unfortunately, I'm not sure how to accomplish this, as the tray is loaded by Obsidian itself and the Obsidian window is not under its control until it does. There is a bootstrapping phase that we cannot currently skip.

**Comment by R4GN3R0** [comment]
> > For your first item, there are two hide options: `Hide on launch` and `Hide taskbar icon`. Those both work for me. Do they not work for you?
> 
> In my case, I have the `Hide on launch` option enabled, and initially, I can observe the Obsidian window along with the loading bar on login. However, it is automatically hidden after a few seconds.
> 
> I suspect it is hidden the moment the Tray is loaded. So, if you have a high number of plugins installed, as I do, or if you have a slow machine, you may see the Obsidian loading bar for a few seconds. While this is not a major inconvenience for me, it can be a bit annoying. I would prefer it to remain hidden from the beginning.
> 
> Unfortunately, I'm not sure how to accomplish this, as the tray is loaded by Obsidian itself and the Obsidian window is not under its control until it does. There is a bootstrapping phase that we cannot currently skip.

Yes, this is what I would like to avoid aswell. The Obsidian loading window after PC's startup.
